pyanaconda/modules/common/containers.py

```diff
@@ -37,6 +37,11 @@
     message_bus=DBus
 )
 
+PayloadContainer = DBusContainer(
+    namespace=PAYLOAD_BASE_NAMESPACE,
```

Comment:
Btw. this container will publish objects at for example `.../Modules/Payload/Base/1`.

Reply:
Ups this shouldn't be here. It's part of the other PR.

Reply:
Fixed. I really should more look on what I'm posting as PR before posting it...